packages/playground/windows/playground-win32/Playground-win32.vcxproj

```diff
@@ -168,34 +168,26 @@
   <ImportGroup Label="ReactNativeWindowsTargets">
     <Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.targets" Condition="Exists('$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.targets')" />
   </ImportGroup>
+  <ItemGroup>
+    <PackageReference Include="boost" Version="1.76.0.0" />
+    <PackageReference Include="Microsoft.Toolkit.Win32.UI.XamlApplication" Version="6.1.3" />
+    <!--
+      WARNING - DO NOT MAKE THE VERSION OF MICROSOFT.UI.XAML BE ANYTHING BUT PRERELEASE.
+      Playground-Win32 is an unpackaged win32 app, and needs to use Microsoft.UI.Xaml from a prerelease,
+      to be able to carry WinUI in-app instead of using the Framework Package
+    -->
+    <!-- <PackageReference Include="Microsoft.UI.Xaml" Version="2.6.1-prerelease.210709001" /> -->
+    <PackageReference Include="$(WinUIPackageName)" Version="$(WinUIPackageVersion)" />
+    <PackageReference Include="Microsoft.Windows.CppWinRT" Version="2.0.211028.7" />
+    <PackageReference Include="Microsoft.VCRTForwarders.140" Version="1.0.2-rc" />
+    <PackageReference Include="ReactNative.Hermes.Windows" Version="0.8.1-ms.5" />
+    <PackageReference Include="$(V8PackageName)" Version="$(V8Version)" Condition="'$(UseV8)' == 'true'" />
```

Comment:
Do you suggest reverting the Playground app to use `packages.config`, or is this a remark about the current VS experience?

Reply:
It's anywhere we have the pattern of:
```xml
 <PackageReference Condition="'..." />
```

Visual Studio will restore, even if the Condition evaluates to false.

Reply:
I'm not sure how many places that is. Not sure reverting the `packages.config` is the right choice, but it would be good to know if we have a workaround.